ocs_ci/framework/pytest_customization/marks.py

```diff
@@ -205,6 +205,10 @@
     reason="Test runs ONLY on OSD or ROSA cluster",
 )
 
+kms_config_required = pytest.mark.skipif(
+    config.ENV_DATA.get("vault") is not True,
```

Comment:
Is ```vault``` already defined?

Reply:
It was not defined under `ENV_DATA`. Changed the skip condition accordingly.